rspamd.service

```
@@ -9,9 +9,39 @@ NonBlocking=true
 ExecStart=/usr/bin/rspamd -c /etc/rspamd/rspamd.conf -f
 ExecReload=/bin/kill -HUP $MAINPID
 User=_rspamd
+ConfigurationDirectory=rspamd
 RuntimeDirectory=rspamd
 RuntimeDirectoryMode=0755
+StateDirectory=rspamd
+StateDirectoryMode=0700
```

```suggestion
StateDirectory=rspamd
StateDirectoryMode=0700
```

This creates `/var/lib/rspamd` owned by the rspamd user with permissions 0700.